.github/ISSUE_TEMPLATE/design_doc.md

```diff
@@ -9,39 +9,35 @@ assignees: ''
 
 *This is for internal use only; if you'd like to open an issue or request a new feature, please open a bug or enhancement issue*
 
-**Section 1: Design details**
+### Instructions
+This document should be filled out prior to embarking on any project that will take more than a couple hours to complete. The goal is to make sure that everyone is on the same page for the functionality and requirements of new features. Therefore, it's important that this is detailed enough to catch any misunderstandings beforehand. For larger projects, it can be useful to first give a high-level sketch, and then go back and fill in the details. For smaller ones, filling the entire thing out at once can be sufficient.  
 
 **Relevant background**
 
 Give a brief (1-2 paragraph) description of the problem/issue that this feature is solving, and how it addresses the issue
 
 **Design overview**
 
-Describe each of the main components of the proposed project
+Give a high-level description of what the proposed project will actually do
 
-**Design list/flowchart**
+**Code mockup**
 
-Provide an overview of how the different pieces of the project will interact with each other, either in a numbered list, bullet point, or brief sketch
+Provide a more detailed description of how the project will implement the above functionality. This should describe the logical flow of the program, detailing how the different parts interact with each other. It should list the specific helper functions that will be created, either with a description of what they'll do, or with psuedo-code for how they'll be implemented. 
 
 **Required inputs**
 
-Provided a description of the required inputs for this project, including column names for dfs, dimensions for image data, prompts for user input, etc
+Provided a description of the required inputs for this project, including column names for dfs, dimensions for image data, prompts for user input, directory structure for loading data, etc
 
 **Output files**
 
-Provide a description of the outputs for this project, including column names for dfs, dimensions for image data, etc. If any plots will be generated, 
-provide (simple) sketches demonstrating the plot type and axes labels. 
+Provide a description of the outputs for this project. If any plots will be generated, provide (simple) sketches demonstrating the plot type and axes labels. 
 
-**Section 2: Implementation details**
+**Timeline**
+Give a rough estimate for how long you think the project will take. In general, it's better to be too conservative rather than too optimistic. 
+- [ ] A couple days
+- [ ] A week
+- [ ] Multiple weeks
```

Comment:
Do you think that for multiple week projects, we should create a github project for it? Such as https://github.com/orgs/angelolab/projects/5? Instead of it being an organization wide project, it would be a repository-based project? This would allow us to tackle several smaller PRs instead of one huge monolithic PR (only if it makes sense however).